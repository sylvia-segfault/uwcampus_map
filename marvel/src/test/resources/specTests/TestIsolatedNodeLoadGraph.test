# Test finding a path on a graph that contains
# an isolated node

LoadGraph graph isolatedNode.tsv
ListNodes graph
# test if G, an isolated node, has any children
ListChildren graph G
# test if I, an isolated node, has any children
ListChildren graph I