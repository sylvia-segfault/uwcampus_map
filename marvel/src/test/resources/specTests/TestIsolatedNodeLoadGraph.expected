# Test finding a path on a graph that contains
# an isolated node

loaded graph graph
graph contains: A B C D E F G I
# test if G, an isolated node, has any children
the children of G in graph are:
# test if I, an isolated node, has any children
the children of I in graph are: